ansible/playbooks/adhoc/uninstall.yml

```diff
@@ -41,6 +41,14 @@
       changed_when: False
       failed_when: False
 
+    - name: Clear etcd configuration data
+      command: etcdctl rm --recursive {{ item }}
+      with_items:
+        - "/registry"
+        - "/cluster.local" #cluster name. default in group_vars/all.yml
```

Comment:
@adamschaub I agree with @eparis 

It would be much appreciated if you could test and send a follow-on PR that incorporates: https://github.com/kubernetes/contrib/blob/master/ansible/group_vars/all.yml#L9-L12